prow/config/jobs/envoy.yaml

```diff
@@ -33,6 +33,23 @@ jobs:
     value: "/home/prow/go/src/istio.io/envoy"
   command: [./ci/do_ci.sh, bazel.release]
 
+periodics:
+- interval: 168h
+  name: update-proxy
+  command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=proxy
+  - "--title=Automator: update envoy@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH"
+  - --labels=auto-merge,release-notes-none
+  - --modifier=update_envoy_dep
```

Comment:
Not familiar with the configuration here, but is `update_envoy_dep` a keyword that points to something? or just a random name?

Reply:
https://github.com/istio/test-infra/tree/master/tools/automator `A segment of text to append to the branch name of the fork for additional context. This option is optional and will default to automator.sh.`